protoc-gen-twirp/generator.go

```diff
@@ -649,6 +651,8 @@ func (t *twirp) generateUtils() {
 	t.P(`  if err != nil {`)
 	t.P(`    return clientError("could not build request", err)`)
 	t.P(`  }`)
+	t.P(`  req.Header.Set("Accept", "application/json")`)
```

Comment:
If the user has set their own `Accept` header (through `WithHTTPRequestHeaders`), we will be overriding it.

We need to either document this in `WithHTTPRequestHeaders` or rearrange things so that the `WithHTTPRequestHeaders` value wins.

I would prefer rearranging things. That way we aren't breaking compatibility in any way.